### PR TITLE
[MWPW-133254] Pricing Summary V3x block

### PR DESCRIPTION
**Pull request by redvelvet511**
https://jira.corp.adobe.com/browse/MWPW-133254

[MWPW-133254]

**Description**
- Option for authors to change CTA color
- Option for authors to add 2 or 3 column
- Option for authors to add a 'details' learn more link

Test **URLs:**
- Before: https://main--express--adobecom.hlx.page/express/drafts/pricing-summary?lighthouse=on
- After: https://mwpw-133254--express--adobecom.hlx.page/express/drafts/pricing-summary?lighthouse=on

### CODE REVIEW

File: express/blocks/pricing-summary/pricing-summary.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+function handleColumn(column) {
+  const header = handleHeader(column);
+  const pricePlan = handlePrice(column);
+  const cta = handleCtas(column);
+  const description = handleDescription(column);
+  const spacer = handleSpacer();
+
+  column.append(header, description, spacer, pricePlan, cta);
+  cleanEmptyParagraphs(column);
```
Comment (qiyundai) [inline express/blocks/pricing-summary/pricing-summary.js:175]: Once we reduce the DOM manipulation for this block, we shouldn't need this.

**Review by qiyundai — CHANGES_REQUESTED** [review]
Left a few points. Overall, I think we over-engineered this block. Happy to have a call to go over together! 😄

**Review by qiyundai — APPROVED** [review]
Unblocking merge but we should follow with test and possible enhancement

**Comment by vhargrave** [comment]
@qiyundai & @redvelvet511 does this also need to get merged to stage?

**Pull request closed without merging**